crates/bitwarden/src/vault/cipher/attachment.rs

```diff
@@ -39,7 +39,7 @@ impl KeyEncryptable<Attachment> for AttachmentView {
             size: self.size,
             size_name: self.size_name,
             file_name: self.file_name.encrypt_with_key(key)?,
-            key: self.key.encrypt_with_key(key)?,
+            key: self.key.map(|k| k.encrypt_with_key(key)).transpose()?,
```

Comment:
Encryptable is implemented for &[u8], so you should be able to do something like this:
```suggestion
            key: self.key.as_deref().encrypt_with_key(key)?,
```